granulate_utils/linux/cgroups/base_cgroup.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import os
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+from granulate_utils.linux.cgroups.exceptions import MissingCgroup, MissingController, SkippedCgroup, UnsupportedCgroup
+
+PID_CGROUPS = Path("/proc/self/cgroup")
+CGROUP_PARENT_PATH = Path("/sys/fs/cgroup")
+
+
+class HIERARCHIES(Enum):
```

Comment:
Is the correct term here "hierarchies" or "subsystem"?

Reply:
its a wrong name - ill replace with `CGROUPS`